cypress/integration/counter_spec.js

```diff
@@ -0,0 +1,43 @@
+describe('ui counter test', () => {
+  const plusButtonClass = '.plus-button';
+  const minusButtonClass = '.minus-button';
+  const countDisplay = '.count-display';
+  const checkCounter = (buttonClass, expectedValue) => {
+      cy.get(buttonClass).click();
+      cy.get(buttonClass).get(countDisplay).should('have.value', expectedValue);
```

Comment:
```suggestion
      cy.get(buttonClass).click()
        .get(countDisplay).should('have.value', expectedValue);
```

공부한 내용중 비동기 처리에 대한 내용을 확인했는데, 아예 처리 자체를 동기적으로 작성하면 비동기적인 테스트코드를 짜지 않아도 되는 테스트었습니다.

Reply:
suggest 해주신 코드와 유사한 코드(기억상)로 테스트 했었을 때 간헐적으로 테스트가 실패해서 고친 부분인데, 이 코드로 테스트 해보니 또 문제 없이 잘 돌아가네요! 그러면 다른 문제가 있었던 건가 싶어서 원인을 찾고 싶은데 기록을 안 해둬서 재현이 안 되는 게 아쉽습니다... 다음부턴 에러 코드도 잘 기록해둬야겠어요. 꼼꼼한 리뷰 감사합니다!!